docs/testlayer.rst

```diff
@@ -76,24 +76,29 @@ ZCMLLayer
 
 We now want a layer that loads up some ZCML from a file. The default
 is ``ftesting.zcml``, but here we'll load a test ``testlayer.zcml``.
+We can also choose to provide extra ZCML features (here, "devmode").
```

Comment:
Are there such docs about "devmode"? 

Maybe just a link to docs about features in general (if there are such things).